### DISCUSSION TITLE
Using async-alpine with Laravel-Livewire

### DISCUSSION THREAD

**Discussion by MariusWaltherGJ** [Accudio/async-alpine#46] [Q&A]
We wanted to use async-alpine to reduce the build size of our application but we encountered a problem with laravel livewire.
After moving the alpine components in async dynamic imports it seems to be the case that the rerendering that livewire is using and triggering also triggers the init of alpine components that use x-load.

This also leads to the error from liverwire that the component is already initilized.

I just wanted to ask if this is a known problem and if there is a solution for it.

**Comment by Accudio** [comment]
@MariusWaltherGJ I don't use Laravel Livewire myself so am not sure on the rendering model and how that may work with Async Alpine.

It's not a problem I'm aware of, I do know others are using Async Alpine with Livewire successfully.

Can you put together a repo with a minimal reproduction of the issue and I can try and look into it?
